CHANGELOG.md

```diff
@@ -97,6 +97,9 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
   - Fixed a race condition with bulk granule delete causing deleted granules to still appear in Elasticsearch. Granules removed via bulk delete should now be removed from Elasticsearch.
 - **CUMULUS-1961**
   - Fixed `activeCollections` query only returning 10 results
+- **CUMULUS-xxx**
+  - Reduced concurrency of `QueueGranules` task, defaulting to `3` and making it
```

Comment:
Can we mention here that it is configurable via `config.concurrency` on the task? I just don't see anywhere else where we advertise available task config?

Reply:
fe27da147

Reply:
In theory, we were working on task pages (https://nasa.github.io/cumulus/docs/tasks) (see Tasks sidebar pull down), but there isn't one for this task.     We could add a quick page, but that's out of scope.